doc/src/vpr/file_formats.rst

```diff
@@ -391,17 +391,34 @@ The ``.param`` statement allows parameters (e.g. primitive modes) to be tagged o
 
 .. note:: ``.param`` statements apply to the previous primitive instantiation.
 
+Parameters can have one of the three available types. Type is inferred from the format in which a parameter is provided.
+
+ * **string**
+    Whenever a parameter value is quoted it is considered to be a string.
```

Comment:
Need to clarify that escaping rules (if any).  For example what is `.param "\""`?  Is that an error?  What is it's behavior?

Reply:
It seems like the external parser is not allowing escaped quote chars. In such case, VPR will throw syntax error there before the flow even gets to the logic responsible for distinguishing between 3 available formats of `.param`.
I modified documentation to highlight this.